packages/bdd-tests/step_definitions/dispenseNotificationSteps.ts

```diff
@@ -0,0 +1,15 @@
+import * as helper from "../util/helper"
+
+import {When} from "@cucumber/cucumber"
+
+When(/^I amend the dispense notification for item (\d+)$/, async function (itemNo, table) {
+  this.resp = await helper.amendDispenseNotification(itemNo, table, this)
+})
+
+When("I withdraw the dispense notification", async function (table) {
```

Comment:
More clashing? Maybe I'm not understanding cucumber properly?

Reply:
You are right - there is a clash.
The tests were originally written using cucumber-jest framework which allowed different definitions for different tests. 
This now wont work, but the tests for dispense do not currently run so it does not show as a clash